### DISCUSSION TITLE
how to config Tailwind CSS to work with rspack.CssExtractRspackPlugin.loader?

### DISCUSSION THREAD

**Discussion by childrentime** [web-infra-dev/rspack#6418] [Q&A]
https://tailwindcss.com/docs/guides/rspack use native css seems to work. But can not work with extract plugin.

**Comment by xc2** [comment]
> https://tailwindcss.com/docs/guides/rspack use native css seems to work. But can not work with extract plugin.

Thank you for your report. I'm closing this thread as duplicated with #6422.
